libbpf-tools/capable.h

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+//
+// Copyright 2022 Sony Group Corporation
+
+#ifndef __CAPABLE_H
+#define __CAPABLE_H
+
+#define TASK_COMM_LEN	16
+
+struct cap_event {
+	__u32	pid;
+	__u32	cap;
+	gid_t	tgid;
+	uid_t	uid;
+	int	audit;
+	int	insetid;
+	char	task[TASK_COMM_LEN];
+};
+
+struct key_t {
+	__u32 pid;
+	__u32 tgid;
+	int user_stack_id;
+	int kern_stack_id;
+};
+
+struct val_t {
```

Comment:
Maybe you can do without a `struct` since it has only field and you are only using it `print_map` to get the command name?

Reply:
Yes, I'm using it to print command name.
I removed redundant `struct val_t`  and instead reused `struct cap_event`.
Pushed the changes. Kindly verify.

Reply:
Hi.

I tested your code with your modifications and it still works fine.
Also, I am only a contributor here (like you), so my review is just to make the code better as I cannot influence the merging decision.

If you modify it again, feel free to ping :).

Best regards.